src/main/java/com/minecolonies/network/messages/ColonyViewWorkOrderMessage.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.network.messages;
+
+import com.minecolonies.colony.CitizenData;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.WorkOrderView;
+import com.minecolonies.colony.workorders.AbstractWorkOrder;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessageHandler;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+
+/**
+ * Add or Update a ColonyView on the client
+ */
+public class ColonyViewWorkOrderMessage implements IMessage, IMessageHandler<ColonyViewWorkOrderMessage, IMessage>
+{
+    private int     colonyId;
+    private int     workOrderId;
+    private int     order;
+    private ByteBuf workOrderBuffer;
+    private workOrderMessages operation;
+
+    public enum workOrderMessages
+    {
+        EDIT,
+        EMPTY
+    }
+
+    /**
+     * Empty public constructor
+    */
+    public ColonyViewWorkOrderMessage()
+    {
+        /**
+         * Intentionally left empty.
+         **/
+    }
+
+    /**
+     * Updates a {@link com.minecolonies.colony.WorkOrderView} of the workOrders
+     *
+     * @param colony  Colony of the citizen
+     * @param workOrder Workorder of the colony to update view
+     */
+    public ColonyViewWorkOrderMessage(Colony colony, AbstractWorkOrder workOrder, int order, workOrderMessages operation)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): order, operation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)